src/main/java/com/powsybl/caseserver/server/CaseController.java

```diff
@@ -4,8 +4,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.caseserver;
+package com.powsybl.caseserver.server;
```

Comment:
To be homogeneous with other repositories : 
Move CaseController to com.powsybl.caseserver 
Rename "services" package to "service". And move the service classes that are in "server" to "service"

Reply:
Done